.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please add system specifics to your personal global gitignore.
See <https://help.github.com/articles/ignoring-files/#create-a-global-gitignore>

Reply:
Oops 👍🏻